### ISSUE TITLE
[iOS]Sound noise and delay on iPhone 14 Plus and Pro

### ISSUE THREAD

**Issue by ryopei** [airsdk/Adobe-Runtime-Support#2314]
## Problem Description

When playing mp3 sound on iPhone14 Plus and iPhone 14 Pro, there is noise and a delay occurs.
The probability of occurrence is more than 50%, but it may not occur easily.
Happened on multiple iPhone 14 Plus and Pro, not iPhone 12 or SE 2nd Gen, iPhone 8.

AIR SDK: 33.1.1.856, 33.1.1.935

Isn't this a hardware-dependent problem for you? You might think so, but it has occurred on at least 3 iPhone 14 devices.

## Steps to Reproduce
The code to reproduce is simple.
```
public class Main extends Sprite {
    public function Main() {
        new Sound(new URLRequest("bgm.mp3")).play()
    }
}
}
```

**Comment by KaseYu** [comment]
I have the same problem with my  iphone 14 pro max. @ajwfrost

**Comment by ryopei** [comment]
Do you have any information?

I'm not sure why it's related, but turning off Auto-Brightness eliminates this problem.
For Auto-Brightness, refer to "About brightness levels" at the following URL.
https://support.apple.com/en-us/HT202613

**Comment by ajwfrost** [comment]
Hi - we'd been investigating this and had come to some similar conclusions...

Interestingly, if you have this adaptive brightness on, then you get a lot of noise/interruptions to the audio when you hold up the phone to a bright light; if you put it somewhere fairly dim, then audio plays back okay.

This is proving to be a difficult one to actually find out the root cause for, I'm a little hesitant to just blame Apple, but it does seem very suspicious! We are looking to see if there are any other options or workarounds we can try to put in place...


thanks

**Comment by raresn** [comment]
I confirm we get the same issue.

**Comment by raresn** [comment]
@ajwfrost any update here? Thank you.

**Comment by ajwfrost** [comment]
Trying to tidy things up a bit -> this one appears to be a duplicate of #2642 (or vice versa) and was fixed in 50.2.4.3 at the start of this year... closing this one now, please re-open if needed.
